shpc/main/modules/base.py

```diff
@@ -375,15 +375,19 @@ def get_module(self, name, container_image=None, keep_path=False):
         return module
 
     def install(
-        self, name, force=False, container_image=None, keep_path=False, **kwargs
+        self,
+        name,
+        allow_reinstall=False,
```

Comment:
As I mentioned above, let's make this `force`
```suggestion
        force=False,
```